api/src/main/resources/messages.properties

```diff
@@ -46,9 +49,16 @@ openhmis.cashier.bill.status=Status
 openhmis.cashier.bill.totalPaid=Total Paid
 openhmis.cashier.bill.totalAmount=Total Amount
 openhmis.cashier.bill.postBill=Post Bill
+openhmis.cashier.bill.postAndPrint=Post and Print
 openhmis.cashier.bill.saveBill=Save Bill
+openhmis.cashier.bill.unsavedChanges=There are unsaved changes.
 openhmis.cashier.bill.printReceipt=Print Receipt
-
+openhmis.cashier.bill.choosePatientErrorMessage=bill needs to be associated with a patient.
```

Comment:
Fix capitalization. I would also change the wording to: "The bill must be associated with a patient."